variables.tf

```diff
@@ -1,24 +1,383 @@
-/********************************************************************
+# ##################################################
+# # Azure and IBM Authentication Variables
+# ##################################################
 
-This file is used to declare ROOT varaibles.
+variable "TF_VERSION" {
```

Comment:
Do we need this variable at module level?

Reply:
Removed variable